lib/createEnum.lua

```diff
@@ -37,13 +37,25 @@ local function createEnum(enumName, variantValues)
 		variants[variantName] = createEnumVariant(enum, variantName, value)
 	end
 
+	local enumItems = {}
+
+	for _, variant in pairs(variants) do
+		enumItems[#enumItems + 1] = variant
+	end
+
 	getmetatable(enum)[typeKey] = "Enum"
 
 	getmetatable(enum).__tostring = function()
 		return enumName
 	end
 
 	getmetatable(enum).__index = function(self, key)
+		if key == "GetEnumItems" then
+			return function()
+				return enumItems
+			end
+		end
```

Comment:
This is kind of a bolt-on solution, we should probably make a table containing the enum variant's methods and explicitly index into it instead of comparing the key with a constant.

Additionally, we should return the same function every time, otherwise `Foo.GetEnumItems ~= Foo.GetEnumItems`!